cpp/util/etcd.cc

```diff
@@ -980,7 +980,8 @@ list<EtcdClient::HostPortPair> SplitHosts(const string& hosts_string) {
   list<EtcdClient::HostPortPair> ret;
   for (const auto& h : hosts) {
     vector<string> hp(util::split(h, ':'));
-    CHECK_EQ(2, hp.size()) << "Invalid host:port string: '" << h << "'";
+    CHECK_EQ(static_cast<size_t>(2), hp.size()) << "Invalid host:port string: '"
```

Comment:
That's really necessary to shut it up?!? How about something like `2UL`, maybe that would work?